core/modules/SConscript

```diff
@@ -56,11 +56,8 @@ allModules = """ccontrol css czar global log mysql obsolete
 # Products
 def pyDistRelative(env, module):
     return os.path.join(env['python_relative_prefix'],"lsst","qserv",module)
-shProducts = { "xrdoss" : {'mods' : """wlog mysql obsolete sql util
-                                       wbase wconfig
-                                       xrdoss wpublish""".split(),
-                           'libs' : """boost_regex boost_signals boost_thread log log4cxx
-                                       mysqlclient_r ssl crypto XrdSsi""".split(),
+shProducts = { "xrdoss" : {'mods' : """global mysql sql obsolete util xrdoss wconfig wpublish""".split(),
```

Comment:
Cool, this looks better now...  I'd say alphabetize these, then looks good to me, thanks!